packages/headless/src/features/commerce/query-set/query-set-actions-loader.ts

```diff
@@ -0,0 +1,24 @@
+import {CommerceEngine} from '../../../app/commerce-engine/commerce-engine';
```

Comment:
I think we should rather create / use / export commerce-prefixed actions.

You can cherry-pick this commit if you want: https://github.com/coveo/ui-kit/commit/6c45fc583193bc9ffa5c0c9e9a3ce7e1fd107743